recipe/meta.yaml

```diff
@@ -17,19 +17,19 @@ requirements:
   build:
     - cmake
     - python
-    - numpy 1.8.*
+    - numpy
     - jasper
-    - libpng >=1.6.32,<1.6.35
-    - libnetcdf 4.5.*
-    - hdf5 1.10.1
+    - libpng
+    - libnetcdf
+    - hdf5
     - ecmwf_grib {{ version }}
   run:
     - python
-    - numpy >=1.8
+    - numpy
     - jasper
-    - libpng >=1.6.32,<1.6.35
-    - libnetcdf >=4.5
-    - hdf5 1.10.1
+    - libpng
+    - libnetcdf
+    - hdf5
     - ecmwf_grib {{ version }}
```

Comment:
should probably be a ``{{ pin_compatible('ecmwf_grib') }}`` expression.  I don't know what the bounds you want are.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-expressions for more details.

Reply:
`ecmwf_grib` must be the exact same version as `python-ecmwf` b/c it is the same package actually, just the python bindings. This recipe should be unified and we should use outputs when we solve the CI situation.

Reply:
Should be {{ pin_compatible('ecmwf_grib', exact=True) }}  then.

When you make a split package for it, use pin_subpackage instead of pin_compatible, but same args.

Reply:
If only the version number matters, not the build string, you should do ``{{ pin_compatible('ecmwf_grib', max_pin='x.x.x.x.x.x.x') }}`` instead.  Too many x's is better than not enough when you want to nail down that version.